release-notes/7.0/preview/api-diff/preview6/Microsoft.WindowsDesktop.App/7.0-preview6_System.Windows.Forms.Design.md

```diff
@@ -0,0 +1,22 @@
+# System.Windows.Forms.Design
+
+``` diff
+ namespace System.Windows.Forms.Design {
+     [ToolboxItemAttribute(false)]
+     public class ComponentEditorForm : Form {
+-        [BrowsableAttribute(false)]
+-        [EditorBrowsableAttribute(1)]
+-        public new event EventHandler AutoSizeChanged;
++        [BrowsableAttribute(false)]
++        [EditorBrowsableAttribute(1)]
++        public new event EventHandler? AutoSizeChanged;
+-        protected virtual void OnSelChangeSelector(object source, TreeViewEventArgs e);
++        protected virtual void OnSelChangeSelector(object? source, TreeViewEventArgs e);
+-        public virtual DialogResult ShowForm(IWin32Window owner);
++        public virtual DialogResult ShowForm(IWin32Window? owner);
+-        public virtual DialogResult ShowForm(IWin32Window owner, int page);
++        public virtual DialogResult ShowForm(IWin32Window? owner, int page);
+     }
```

Comment:
This doesn't look readable - attributes didn't change, only NRT. The diff should look like this:
```diff
     public class ComponentEditorForm : Form {
         [BrowsableAttribute(false)]
         [EditorBrowsableAttribute(1)]
-        public new event EventHandler AutoSizeChanged;
+        public new event EventHandler? AutoSizeChanged;
-        protected virtual void OnSelChangeSelector(object source, TreeViewEventArgs e);
+        protected virtual void OnSelChangeSelector(object? source, TreeViewEventArgs e);
-        public virtual DialogResult ShowForm(IWin32Window owner);
+        public virtual DialogResult ShowForm(IWin32Window? owner);
-        public virtual DialogResult ShowForm(IWin32Window owner, int page);
+        public virtual DialogResult ShowForm(IWin32Window? owner, int page);
     }
```